src/Animal/Animal.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './Animal.css';
+
+export default function Animal({ animal: { name, type, says } }) {
```

Comment:
you don't need the key of animal here. 
instead it could be `{ animal } ` or `{ name, type, says, top, left }